moveit_core/collision_detection/include/moveit/collision_detection/test_collision_common.h

```diff
@@ -469,7 +469,7 @@ TYPED_TEST_P(CollisionDetectorTest, TestCollisionMapAdditionSpeed)
   ros::WallTime start = ros::WallTime::now();
   this->cworld_->getWorld()->addToObject("map", shapes, poses);
   double t = (ros::WallTime::now() - start).toSec();
-  EXPECT_GE(1.0, t);
+  EXPECT_GE(5.0, t);
```

Comment:
Why do you need to change this? Adding objects to the world takes more than a second now?

Reply:
Yes, I haven't looked closer into this but with bullet I need around 3 seconds.

Reply:
This needs to be investigated! Even a second is _really_ long...

Reply:
But these are **10000** shapes, not a single object? So a single object is less than 1 ms...

Reply:
Ok, this is calming me down. Nevertheless, an increase from <1s to >3s is strange and should be understood. Please investigate at some point in future and leave a TODO here for now.

Reply:
is it 10,000 times the same shape/mesh? Then maybe the bullet part creates copies of the mesh while the fcl one has a cache (it just stores the shape_ptr I think and should be even faster ...). Looking at the memory usage should answer this question easily (if not increase number/mesh size)

Reply:
Yes it is the same shape/mesh. FCL definitely is much faster, but the memory usage of both programs seems similar (see below). The reason must be something else, I added the ToDo and will investigate at a later point.

FCL:
![image](https://user-images.githubusercontent.com/19279526/61285324-02dbd980-a7c1-11e9-9aaf-41667ab43636.png)

Bullet:
![image](https://user-images.githubusercontent.com/19279526/61285574-7aaa0400-a7c1-11e9-8484-1ac1a1f5e0d9.png)